src/components/organims/welcomeByOrg.js

```diff
@@ -0,0 +1,141 @@
+import React from 'react';
+import {Redirect, navigate} from '@reach/router';
+import AppMinimal from '../layout/AppMinimal';
+import Seo from '../molecules/Seo';
+import Button from '../atoms/Button';
+import styles from './welcome.module.css';
+import gql from 'graphql-tag';
+
+const getOrgInfo = gql`
+  query($slug: String!) {
+    organizationWithSlug(slug: $slug) {
+      slug,
+      name,
+      pictureUrl,
+    }
+  }
+`;
+
+export class WelcomeByOrg extends React.Component{
+  constructor(props){
+    super(props);
+    this.state = {
+      title: 'Your organization name',
+      description: 'Your organization description',
+      details: 'Your organization details',
+      email: 'name.org',
+      pictureUrl: '',
+      hasDownloaded: false,
+    }
+  }
+  componentWillMount(){
+    const { data, organizationSlug } = this.props;
+    const parsedUrl = new URL(window.location.href);
+    // If url contain ?dl=true just dl the file too !
+    if(parsedUrl && parsedUrl.searchParams.get('dl')){
+      window.location= data.button_url.url;
+    }
+  }
+  componentDidMount(){
+    const { organizationSlug, client, data } = this.props;
+    client
+    .query({
+      query: getOrgInfo,
+      variables: { slug: organizationSlug}
+    })
+    .then(res=>{
+      const {name, pictureUrl, slug} = res.data.organizationWithSlug;
+      if(!name || !slug || ! pictureUrl) {
+        return this.setState({
+          unreachable: true
+        })
+      }
+      this.setState({
+        title: data.maintitle.text.replace('{{organizationName}}', name),
+        description: data.description.text.replace('{{organizationName}}', name),
+        details: data.details.text.replace('{{organizationName}}', name),
+        email:  data.second_step_text.text.replace('{{organizationName}}', slug),
+        pictureUrl,
+        unreachable: false,
+      });
+    })
+    .catch(err=>console.error(err))
+  }
+  handleClickDownload(){
+    const{data} = this.props;
+    window.open(data.button_url.url);
+    this.setState({
+      hasDownloaded: true
+    })
+  }
+  render(){
+    const { data } = this.props;
+    // Separeted into two step, the second is after user as clicked download.
+    if(this.state.unreachable){
+      return <Redirect noThrow to='/404'/>;
+    }
+    return !this.state.hasDownloaded ? (
+      <AppMinimal>
```

Comment:
Saw some differences bu i have not checked in detail though:
<img width="1097" alt="Screenshot 2019-12-02 11 59 50" src="https://user-images.githubusercontent.com/687961/69954369-690c9300-14fb-11ea-889f-b42c21e9dcda.png">

Reply:
Moreover, looks like the workding is still static: `Acme Corp`

Reply:
Yes the content is not the right one, this have to changed inside prismic 👍

Reply:
FYI still have not been updated as of today